test/sor.test.ts

```diff
@@ -55,7 +58,7 @@ describe('SmartOrderRouter', () => {
         );
 
         const swapOptions: SwapOptions = {
-            block: 16900000n,
+            block: 17473810n,
```

Comment:
I believe this might be because we're not handling protocol fees. 
With the old block number the path is using a CS V1 pool which is in recovery mode at that block. With the updated block number it uses a CS V3 pool which is not in recovery mode. I'll check out if I can add whats required.

Reply:
Was actually an issue with the amp. I proposed a fix in #81.